README.md

```diff
@@ -2,136 +2,25 @@
 
 Visit the website at: https://www.opensuse.org/
 
-# Run the project locally
+## Getting Started
 
-1 - Install all the dependencies:
+First, run the development server:
 
-`npm install`
+```bash
+npm run dev
+# or
+yarn dev
+# or
+pnpm dev
+# or
+bun dev
+```
 
-2 - Gulp is our build system and task manager. It will take care of watching for changes to re-build the assets, as well as launching a local test server:
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-`npx gulp`
+## Learn More
 
-3 - Visit http://localhost:8000
+To learn more about Next.js, take a look at the following resources:
 
-# CONTRIBUTING
-
-## Do you want to contribute by providing translations for the new openSUSE website?
-
-If yes is your answer, then you will make thousands of users happy to have the openSUSE landing page in their language.
-
-The openSUSE translations are provided by https://l10n.opensuse.org/. You can log in with your openSUSE account there.
-
-For introducing a new language, you need to continue reading.
```

Comment:
Is this no longer relevant? How will localization be handled going forward?

Reply:
Localization is still something I'd like to get to. I experimented with Next.js' i18n system today and unfortunately their implementation is a little lacking, in that their translation system is incompatible with statically generated sites. That could be a dealbreaker for this software stack.